jinseok/middleware/auth.js

```diff
@@ -0,0 +1,20 @@
+const jwt = require('jsonwebtoken');
+const jwtSecret = process.env.JWT_SECRET;
+
+const validateToken = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
+
+    const decoded = jwt.verify(JSON.parse(token), jwtSecret);
```

Comment:
```suggestion
    const decoded = await jwt.verify(token, process.env.JWT_SECRET);
```
json parse가 꼭 필요한가요?